specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/bms.json

```diff
@@ -8609,6 +8609,11 @@
           "format": "int32",
           "description": "Soft delete retention period in days",
           "type": "integer"
+        },
+        "vaultId": {
```

Comment:
Why change in old API version?

Reply:
@pratikjoshi14 this is to fix swagger correctness issue - https://openapihub.azure-devex-tools.com/amekpis/correctness/detail?errorId=655CA4E4-72C9-4E51-90A5-569420E0B8EF